knack/arguments.py

```diff
@@ -87,63 +124,82 @@ def get_cli_argument(self, command, name):
 
 
 class ArgumentsContext(object):
-    def __init__(self, command_loader, command):
+    def __init__(self, command_loader, command_scope):
+        """ Context manager to register arguments
+
+        :param command_loader: The command loader that arguments should be registered into
+        :type command_loader: knack.commands.CLICommandsLoader
+        :param command_scope: The command scope this context manager should have
```

Comment:
Rec: "The scope to which arguments in this context apply. More specific scopes will override less specific scopes in the event of a conflict."

Reply:
done.